pandas/io/excel/_base.py

```diff
@@ -101,61 +101,61 @@
     List of column names to use. If file contains no header row,
-    then you should explicitly pass header=None.
+    then you should explicitly pass ``header=None``.
 index_col : int, str, list of int, default None
-    Column (0-indexed) to use as the row labels of the DataFrame.
+    Column (0-indexed) to use as the row labels of the ``DataFrame``.
     Pass None if there is no such column.  If a list is passed,
     those columns will be combined into a ``MultiIndex``.  If a
-    subset of data is selected with ``usecols``, index_col
+    subset of data is selected with ``usecols``, ``index_col``
     is based on the subset.
 
     Missing values will be forward filled to allow roundtripping with
     ``to_excel`` for ``merged_cells=True``. To avoid forward filling the
     missing values use ``set_index`` after reading the data instead of
     ``index_col``.
 usecols : str, list-like, or callable, default None
-    * If None, then parse all columns.
-    * If str, then indicates comma separated list of Excel column letters
+    * If ``None``, then parse all columns.
+    * If ``str``, then indicates comma separated list of Excel column letters
       and column ranges (e.g. "A:E" or "A,C,E:F"). Ranges are inclusive of
       both sides.
-    * If list of int, then indicates list of column numbers to be parsed
+    * If list of ``int``, then indicates list of column numbers to be parsed
       (0-indexed).
-    * If list of string, then indicates list of column names to be parsed.
+    * If list of ``str``, then indicates list of column names to be parsed.
     * If callable, then evaluate each column name against it and parse the
       column if the callable returns ``True``.
 
     Returns a subset of the columns according to behavior above.
 dtype : Type name or dict of column -> type, default None
-    Data type for data or columns. E.g. {{'a': np.float64, 'b': np.int32}}
+    Data type for data or columns. E.g. ``{'a': np.float64, 'b': np.int32}``
     Use `object` to preserve data as stored in Excel and not interpret dtype.
     If converters are specified, they will be applied INSTEAD
-    of dtype conversion.
+    of ``dtype`` conversion.
 engine : str, default None
-    If io is not a buffer or path, this must be set to identify io.
-    Supported engines: "xlrd", "openpyxl", "odf", "pyxlsb".
+    If ``io`` is not a buffer or path, this must be set to identify ``io``.
+    Supported engines: ``"xlrd"``, ``"openpyxl"``, ``"odf"``, ``"pyxlsb"``.
     Engine compatibility :
 
-    - "xlrd" supports old-style Excel files (.xls).
-    - "openpyxl" supports newer Excel file formats.
-    - "odf" supports OpenDocument file formats (.odf, .ods, .odt).
-    - "pyxlsb" supports Binary Excel files.
+    - ``"xlrd"`` supports old-style Excel files (.xls).
+    - ``"openpyxl"`` supports newer Excel file formats.
+    - ``"odf"`` supports OpenDocument file formats (.odf, .ods, .odt).
+    - ``"pyxlsb"`` supports Binary Excel files.
```

Comment:
I would suggest no double quotes here (so just e.g. ` ``xlrd`` `). We can just be referring to the engine itself, rather than the string argument.

Reply:
Good call!  I'll make sure to change that in the commit I'm working on.